src/common/syncjournaldb.cpp

```diff
@@ -192,6 +192,27 @@ bool SyncJournalDb::maybeMigrateDb(const QString &localPath, const QString &abso
     return true;
 }
 
+bool SyncJournalDb::checkVersion(const QString &dbFilePath)
```

Comment:
Why is naming things so hard, I suggested checkVersion but I'm not happy 🙈 
Maybe checkDowngrade? In which case we would need to invert the condition.

Reply:
Done, but is the version check correct? Meaning, do we only want to check the major version?

Reply:
As minor version changes are generally mean that there where no breaking changes I'd hope comparing the major version is sufficient.